notebooks/233-blip-visual-language-processing/233-blip-quantize-nncf.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        inputs = processor(example["image"], example['question'], return_tensors="pt")
</pre><p>as I know processor return_tensors  also support 'np' value for getting numpy arrays, there is no need to use torch and convert it to numpy explicitly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>